ansible/roles/common/templates/initialize.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 set -e
 
-#sudo -u postgres createuser --superuser liquid
+
+sudo -u postgres psql postgres -tAc "SELECT 1 FROM pg_roles WHERE rolname='liquid'" | grep -q 1 || sudo -u postgres createuser --superuser liquid
```

Comment:
That's a great idea! Can we refactor it to be more readable, using an `if` perhaps?